roles/k3s/master/tasks/main.yml

```diff
@@ -13,51 +13,11 @@
   args:
     warn: false  # The ansible systemd module does not support reset-failed
 
```

Comment:
Do we need to include the tasks from `kured.yml` here?

Reply:
I did messed up there, I think working late in December had my brain fuzzy and forgotten about the important task.